elements/pfe-card/pfe-card.scss

```diff
@@ -231,6 +231,12 @@ $size-small: (
       --pfe-card__overflow--MarginBottom: calc(#{pfe-local(spacing--vertical)} * -1);
     }
 
+    &:is(.has-header){
+      ::slotted(img[overflow~="top"]){
```

Comment:
Looks fine. Arguably the spacing should be on the header not the image, but i'm not overly bothered

Reply:
Yeah I went back and forth on this and as the heading already had what looked like spacing added onto the bottom of it and the image was the element ignoring that I opted to add the css onto the image itself.